org.eclipse.scanning.device.ui/src/org/eclipse/scanning/device/ui/points/ValidateResultsView.java

```diff
@@ -0,0 +1,239 @@
+			styledString.append("' at ");
+			styledString.append(new Date(System.currentTimeMillis()).toString(), StyledString.COUNTER_STYLER);
+			styledString.append("\n");
+        	
+			styledString.append(appendResultsToStyledString(results));
+
+            setThreadSafeText(text, styledString);
+            return; 
+
+		} catch (Exception ne) {
+			logger.error("Cannot create validation results", ne);
+			if (ne.getMessage()!=null) {
+				setThreadSafeText(text, ne.getMessage());
+			} else {
+				setThreadSafeText(text, ne.toString());
+			}
+		}
+	}
+	
+	/**
+	 * Turn the results into a styled string for display to user, highlighting key values
+	 * @param results The ValidationResults
+	 * @return a StyledString to display on the view
+	 */
+	private StyledString appendResultsToStyledString(ValidateResults results) {
+		StyledString styledString = new StyledString();
+		if (results.getResults() != null) {
+			if (results.getResults() instanceof String) {
+				// Parse out key information from the raw PVStructure string
+				String resultString = (String)results.getResults();
+	
+				styledString.append(resultString);
+				
+				int durationIndex = resultString.indexOf("double duration");
+				if (durationIndex != -1) {
+					int indexOfNewLine = resultString.substring(durationIndex).indexOf("\n");
+					if (indexOfNewLine == -1) {
+						indexOfNewLine = resultString.substring(durationIndex).length() - 1;
+					}
+			    	styledString.setStyle(durationIndex + "double".length(), indexOfNewLine - "double".length(), FontStyler.BOLD);		    	
+				}
+				
+				int axesToMoveIndex = resultString.indexOf("string[] axesToMove");
+				if (axesToMoveIndex != -1) {
+					int indexOfNewLine = resultString.substring(axesToMoveIndex).indexOf("\n");
+					if (indexOfNewLine == -1) {
+						indexOfNewLine = resultString.substring(axesToMoveIndex).length();
+					}
+			    	styledString.setStyle(axesToMoveIndex + "string[]".length(), indexOfNewLine - "string[]".length(), FontStyler.BOLD);		    	
+				}
+				
+			} else {
+				// not a string, just print out the results object
+				 styledString.append(results.getResults().toString());
+			}
+		}
+		
+		return styledString;
+		
+	}
+
+	/**
+	 * Set the text of the display from a String object
+	 * @param text The StyledText object on the display to set
+	 * @param string The text with which to set the display to
+	 */
+	private void setThreadSafeText(StyledText text, String string) {
+		setThreadSafeText(text, new StyledString(string));
+	}
+	
+	/**
+	 * Set the text of the display from a StyledString object
+	 * @param text The StyledText object on the display to set
+	 * @param styledString The text with which to set the display to in StyledString format
+	 */
+	private void setThreadSafeText(StyledText text, StyledString styledString) {
+		if (text.isDisposed()) return;
+    	text.getDisplay().syncExec(new Runnable() {
+    		public void run() {
+    			if (text.isDisposed()) return;

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00122)